jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <j-tray-plugin>
+    <v-row>
+      <j-docs-link :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#metadata-viewer'">View metadata.</j-docs-link>

```

Comment:
@kecnry , how does the `#` syntax work? Does it rely on Sphinx ref or just the plain text section header?

Reply:
I believe the html anchors are generated from the Sphinx refs, so this should be good!